administration/src/application/components/forms/ApplicationForm.tsx

```diff
@@ -1,10 +1,19 @@
 import { SetState, useUpdateStateCallback } from '../../useUpdateStateCallback'
 import { ApplicationType, BlueCardApplicationInput, BlueCardEntitlementType } from '../../../generated/graphql'
 import SwitchDisplay from '../SwitchDisplay'
-import { FormControl, FormControlLabel, FormLabel, Radio, RadioGroup } from '@mui/material'
+import { Button, Checkbox, FormControl, FormControlLabel, FormGroup, FormLabel, Radio, RadioGroup } from '@mui/material'
 import { Form } from '../../FormType'
 import StandardEntitlementForm, { StandardEntitlementFormState } from './StandardEntitlementForm'
 import PersonalDataForm, { PersonalDataFormState } from './PersonalDataForm'
+import BasicDialog from '../BasicDialog'
+import { dataPrivacyBaseHeadline, dataPrivacyBaseText } from '../../../constants/dataPrivacyBase'
+import styled from 'styled-components'
+
+const PrivacyPolicyButton = styled(Button)`
```

Comment:
@michael-markl are we actually using styled-components? I can not remember using it.

Reply:
I think we do for blueprint. For mui, there might be some other options (and we might be inconsistent here)...
See https://mui.com/material-ui/guides/styled-engine/

Reply:
No, we don't, I think we can ignore this for now